.github/workflows/build_and_test.yml

```diff
@@ -25,17 +26,18 @@ jobs:
         just script/unix/ci-test
 
   build-macos:
-    runs-on: macos-11
+    runs-on: macos-13
```

Comment:
if choosing to support apple silicon could add this to a matrix like

Reply:
Is this comment complete? I have no Apple hardware at hand, feel free to suggest a good matrix for a modern user!

Reply:
I read into the docs of cibuildwheel and it looks like we are already configured correctly to build the right wheels, I think we just need to use the GitHub M1 runner to correctly build all the variants, so I've made that change now.

Reply:
Thanks, I discovered on the docs that building all `x86_64`, `arm64` and `universal2` is useless, because `pip`will pick the more specific wheel. Given the small size of the lib perhaps I'll switch to `universal2` only in the future.